docker-compose.yml

```diff
@@ -294,6 +294,19 @@ services:
       - databases
     tmpfs: /var/lib/mariadb
 
+  mariadb-11-0:
+    image: mariadb:11
+    restart: unless-stopped
+    environment:
+      MYSQL_USER: root
+      MYSQL_ROOT_PASSWORD: prisma
+      MYSQL_DATABASE: prisma
+    ports:
+      - '3308:3306'
```

Comment:
Should we use a different port than MariaDB 10 container?

Reply:
I intentionally kept it the same port for now because I haven't _really_ integrated it on CI. I would need to add a new test-kit version for that, update engineer etc. For now, it's just to test it locally, under the existing "mariadb" (🤡) version